### PR TITLE
Add method 'light/get_profile_units'

### LINKED ISSUE #118: Proposal: implement a method in WS API to get someone profile
Actually anyone can post their public profile from the wallet or byteball.co using the message "profile". It would be great if we could also get the complete user profile directly from the WS API. Now there no easy way for a light wallet to get someone complete profile. 

Ideally the method should make sense of the complete profile and allow user to update or delete any field(s). In a way that when an user post this profile payload:
```
{
  name: 'Fabien',
  location: 'Thailand',
  email: 'fabien@domain.com'
}
```
then later post:
```
{
  location: 'France',
  email: null,
  age: 30
}
```
With `get_profile` we would get this result:
```
{
  name: 'Fabien',
  location: 'France',
  age: 30
}
```

With this change Byteball 3p apps could leverage this data to enable social features.

### PR DESCRIPTION
**Pull request by bonustrack**
Fix #118

### CODE REVIEW

File: network.js [hunk 1]
```diff
@@ -2706,6 +2706,34 @@ function handleRequest(ws, tag, command, params){
 			});
 			break;
 
+		case 'light/get_profiles_units':
```
Comment (tonyofbyteball) [inline network.js]: get_profile_units

File: network.js [hunk 2]
```diff
@@ -2706,6 +2706,34 @@ function handleRequest(ws, tag, command, params){
...
+			if (!ValidationUtils.isNonemptyArray(addresses))
+				return sendErrorResponse(ws, tag, "addresses must be non-empty array");
+			if (!addresses.every(ValidationUtils.isValidAddress))
+				return sendErrorResponse(ws, tag, "some addresses are not valid");
+			if (addresses.length > 100)
+				return sendErrorResponse(ws, tag, "too many addresses");
+			async.map(addresses, function(address, cb) {
+				db.query(
+					"SELECT unit FROM messages JOIN unit_authors USING(unit) \n\
+					JOIN units USING(unit) WHERE app='profile' AND address=? \n\
```
Comment (tonyofbyteball) [inline network.js]: Why not use address IN(?) and fetch all in one query?

File: network.js [hunk 3]
```diff
@@ -2706,6 +2706,34 @@ function handleRequest(ws, tag, command, params){
...
+					ORDER BY main_chain_index DESC", [address], function(rows) {
```
Comment (tonyofbyteball) [inline network.js]: ASC would be better for future merging of profiles

**Comment by bonustrack** [comment]
I made the changes requested @tonyofbyteball.

**Comment by tonyofbyteball** [comment]
Please resolve conflicts, otherwise looks good.

**Comment by bonustrack** [comment]
Thx, done @tonyofbyteball

**Pull request merged**